victim-loss-identifier/src/fraud.nft.order.processing.ts

```diff
@@ -29,11 +29,11 @@ function hasBuyerOrNftExchangeTransferredToSeller(
     const isTransferSingleEvent = log.topics[0] === utils.id("TransferSingle(address,address,address,uint256,uint256)");
     return (
       (isTransferEvent &&
+        log.topics[2].includes(sellerAddress.slice(2)) &&
         (log.topics[1].includes(buyerAddress.slice(2)) ||
-          (Object.values(EXCHANGE_CONTRACT_ADDRESSES).some((exchangeAddress) =>
+          Object.values(EXCHANGE_CONTRACT_ADDRESSES).some((exchangeAddress) =>
             log.topics[1].includes(exchangeAddress.slice(2))
-          ) &&
-            log.topics[2].includes(sellerAddress.slice(2))))) ||
+          ))) ||
       (isTransferSingleEvent &&
         log.topics[2].includes(buyerAddress.slice(2)) &&
```

Comment:
I'd argue it is not necessary, as even in the case of a simple `Transfer` event, that refers to an ERC20 (WETH) being transferred rather than a ERC721. (That's what the FP that I was seeing last week was about)

I suggest leaving it as it is, at least for now.